src/coreclr/jit/gentree.cpp

```diff
@@ -19348,6 +19353,19 @@ GenTree* Compiler::gtNewSimdAbsNode(var_types type, GenTree* op1, CorInfoType si
         assert(compIsaSupportedDebugOnly(InstructionSet_AVX2));
         intrinsic = NI_AVX2_Abs;
     }
+    else if (simdSize == 64)
+    {
+        if (simdBaseType == TYP_INT)
+        {
+            assert(compIsaSupportedDebugOnly(InstructionSet_AVX512F));
+            intrinsic = NI_AVX512F_Abs;
+        }
+        else if (varTypeIsSmall(simdBaseType))
+        {
+            assert(compIsaSupportedDebugOnly(InstructionSet_AVX512BW));
+            intrinsic = NI_AVX512BW_Abs;
+        }
```

Comment:
With `unsigned` and `TYP_LONG` being handled above, I think this could be:
```suggestion
        else
        {
            assert(varTypeIsSmall(simdBaseType));
            assert(compIsaSupportedDebugOnly(InstructionSet_AVX512BW));
            intrinsic = NI_AVX512BW_Abs;
        }
```